docs/workloads/azure/data/etl_pipelines/silver_data_azure.md

```diff
@@ -12,10 +12,9 @@ keywords:
   - databricks
 ---
 
-Silver processing represents the 'Bronze to Silver' layer transformations. It accepts raw data
-(e.g. CSV files) as an input and converts it into a more structured delta format.
-Azure Data Lake Storage (ADLS Gen2) has been chosen as both the source and destination system for
-our data processing.
+Silver processing represents the 'Bronze to Silver' layer transformations. It accepts raw data of various formats
```

Comment:
Probably in this section I might highlight the aims of the silver layer - people may or may not know what it's used for, so talking about cleansing, conformance and data quality is probably good?